src/components/pages/Manage/Manage.tsx

```diff
@@ -1,55 +1,32 @@
 import { Tooltip } from '@src/components/data-display';
-import { Loader } from '@src/components/layout';
-import { useArNSState, useModalState, useWalletState } from '@src/state';
-import { Progress, Table } from 'antd';
-import { useEffect, useState } from 'react';
-import { Link, useLocation, useNavigate, useParams } from 'react-router-dom';
-
-import { useWalletANTs, useWalletDomains } from '../../../hooks';
-import { ManageTable } from '../../../types';
-import { MANAGE_TABLE_NAMES } from '../../../types';
+import { ArweaveTransactionID } from '@src/services/arweave/ArweaveTransactionID';
 import {
-  doAntsRequireUpdate,
-  getCustomPaginationButtons,
-} from '../../../utils';
-import { CodeSandboxIcon, NotebookIcon, RefreshIcon } from '../../icons';
+  dispatchArNSUpdate,
+  useArNSState,
+  useGlobalState,
+  useModalState,
+  useWalletState,
+} from '@src/state';
+import eventEmitter from '@src/utils/events';
+import { useState } from 'react';
+
+import { doAntsRequireUpdate } from '../../../utils';
+import DomainsTable from '../../data-display/tables/DomainsTable';
+import { RefreshIcon, SearchIcon } from '../../icons';
 import './styles.css';
 
 function Manage() {
-  const navigate = useNavigate();
-  const { path } = useParams();
-  const location = useLocation();
-  const {
-    columns: antColumns,
-    rows: antRows,
-    refresh: refreshANTs,
-  } = useWalletANTs();
-  const {
-    columns: domainColumns,
-    rows: domainRows,
-    refresh: refreshDomains,
-  } = useWalletDomains();
-  const [{ percentLoaded: percent, loading: tableLoading, ants, luaSourceTx }] =
-    useArNSState();
+  const [{ ioProcessId }] = useGlobalState();
+  const [
+    { loading: loadingArnsState, domains, ants, luaSourceTx, arnsEmitter },
+    dispatchArNSState,
+  ] = useArNSState();
   const [{ walletAddress }] = useWalletState();
   const [, dispatchModalState] = useModalState();
-
-  const [tablePage, setTablePage] = useState<number>(1);
-
-  useEffect(() => {
-    if (!path) {
-      navigate('names');
-      return;
-    }
-    setTablePage(1);
-  }, [path]);
-
-  function updatePage(page: number) {
-    setTablePage(page);
-  }
+  const [search, setSearch] = useState<string>('');
 
   return (
-    <div className="page" style={{ overflow: 'auto' }}>
+    <div className="page" style={{ overflow: 'auto', paddingTop: '0px' }}>
```

Comment:
maybe go ahead and switch this to tailwind and use `className="page pt-0 overflow-auto"`